manifests/e/Eliot/UEExplorer/1.3.1/Eliot.UEExplorer.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Eliot.UEExplorer
+PackageVersion: 1.3.1
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://eliotvu.com/updates/ue-explorer.1.3.1.0.msi
+  InstallerSha256: F1AF3EF606C7E4B0A8694ABA0943840A2C815331FC232FA1904F4F10444A8CA9
+  ProductCode: '{E29ADE45-052E-4CD4-9B13-0D106DA1CB49}'

```

```suggestion
  ProductCode: '{E29ADE45-052E-4CD4-9B13-0D106DA1CB49}'
  AppsAndFeaturesEntries:
    - DisplayVersion: 1.3.2
```
WinGet requires the metadata of the package to be consistent with Registry values. If you market the package as `1.3.1`, we can use this in the PackageVersion field which will be the version shown to the user but provide AppsAndFeaturesEntries in the manifest and explicitly provide the `DisplayVersion` (reg value) for WinGet to match upgrades.

Please note that this is only a workaround as this way has the potential to cause an infinite upgrade loop for the user in subsequent releases (or we can have a case where WinGet doesn't upgrade at all to a future `1.3.2` release as this version's DisplayVersion is already `1.3.2`). We would probably need to remove this version with the next release.

My personal recommendation would be to release a new setup fixing this DisplayVersion string written to registry, or if you want this package to be available right now then I'd say to remove this version from the repository when you submit a new version to the repo.

---

Thanks, I have updated the manifest and the installer (at the URL) with the applied changes as per your recommendation.

I also changed the publisher from "Eliot" to "EliotVU" to be consistent in all places.
Just noticed, does this mean I'll also have to rename the /Eliot/ folder hmm..

---

That's not a strict rule. The goal is to make a unique package ID and in WinGet it's generally in the form of `Publisher.PackageName`. It's up to you if you want to do it for consistency.

---

The file path, file name, and PackageIdentifier for all 3 files now match Eliot and UEExplorer.